test/pkg/environment/common/expectations.go

```diff
@@ -49,6 +49,20 @@ import (
 	coreresources "sigs.k8s.io/karpenter/pkg/utils/resources"
 )
 
+const (
+	DefaultControllerNamespace = "kube-system"
+	DefaultDeploymentName      = "karpenter"
+)
```

Comment:
Do these need to be exported? Is `Default` prefix justified? Maybe just `karpenterNamespace` and `karpenterDeploymentName`?

Reply:
Please address or respond